packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -130,6 +133,8 @@ const MenuItem: VibeComponent<MenuItemProps | MenuItemTitleComponentProps> & {
           className={className}
           disabled={disabled}
           selected={selected}
+          tooltipId={tooltipProps?.id}
+          shouldShowTooltip={!!shouldShowTooltip}
```

Comment:
In order to simplify it it might be clearer to add `aria-disabled` and `aria-describedby` props instead of the tooltip specific props so that the logic can be done here.